tests/data/codelist/illegal_chars/char_in_external_repo/nomenclature.yaml

```diff
@@ -0,0 +1,11 @@
+dimensions:
+  - variable
+repositories:
+  common-definitions:
+    url: https://github.com/IAMconsortium/common-definitions.git/
+    hash: 3b75bb9
+definitions:
+  variable:
+    repository:
+      - common-definitions
+illegal_characters: ['"' , ";"]  # these are known to be present in common-definitions variables
```

Comment:
I'm not sure that this should be a project specific configuration. I would just extend the `forbidden` list in the illegal character check.
Maybe there's a good reason to make it project specific though.